src/server/appHandler.js

```diff
@@ -69,7 +69,7 @@ export default function (req, res) {
   .catch(err => {
     res.errors = [err]
     res.setHeader('Content-Type', 'text/plain')
-    const state = parse(req.url, true).query.verboseErrorPage
+    const state = parse(req.originalUrl, true).query.verboseErrorPage
```

Comment:
I was getting an error in this branch (in addition to the error that is being caught by this branch) because the req object didn't have a url key. It did have an originalUrl key, which fixed the error.